ui/src/app/pages/login/account-creation/account-creation-survey.tsx

```diff
@@ -36,27 +33,21 @@ export const AccountCreationSurvey = withProfileErrorModal({title: 'Error creati
   async createAccount(profile, captchaToken): Promise<Profile> {
     const {invitationKey, termsOfServiceVersion, onComplete} = this.props;
 
-    try {
-      const newProfile = await profileApi().createAccount({
-        profile: profile,
-        captchaVerificationToken: captchaToken,
-        invitationKey: invitationKey,
-        termsOfServiceVersion: termsOfServiceVersion
-      });
-      onComplete(newProfile);
-      return newProfile;
-    } catch (error) {
-      reportError(error);
-      const errorResponse = await convertAPIError(error);
-      this.props.showProfileErrorModal(errorResponse.message);
```

Comment:
do we not want to do this anymore for the account creation survey?

Reply:
Does this clarify? 

https://github.com/all-of-us/workbench/pull/4637/files#diff-b357383f9ad2e72a406e31a660428141a50eb36f8e71765348e9b4baa4697273R48

and

https://github.com/all-of-us/workbench/pull/4637/files#diff-ba120b5cb4c891762187a4176b9cfa087174b85bff8c99291dd6355f4df71beeR170

Reply:
Oh, I think I see. We pushed the line that displays the error down into the DemographicSurvey component whereas before we'd had each component kind of handle their own thing

Reply:
Precisely, which meant we could fully test the functionality in one place instead of two places.